examples/third_party_integration/flux/flux_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flux
+
+import (
+	"context"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+	"sigs.k8s.io/e2e-framework/third_party/flux"
+	"testing"
+	"time"
+)
+
+var curDir, _ = os.Getwd()
+
+func TestFluxRepoWorkflow(t *testing.T) {
+	gitRepoName := "hello-world"
+	kustomizationName := "hello-world"
+
+	feature := features.New("Install resources by flux").
+		Setup(func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
```

Comment:
Suggestion: this setup/teardown probably belongs in a test environment setup/finish block in TestMain.

Reply:
That is a great point, agree. It should go to setup block.